org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/refactoring/IRefactoringUpdateAcceptor.java

```diff
@@ -32,7 +33,7 @@
 
 	StatusWrapper getRefactoringStatus();
 	
-	IRefactoringDocument getDocument(URI resourceURI);
+	@Nullable IRefactoringDocument getDocument(URI resourceURI);
```

Comment:
One remark for this one: xtext.util.internal.Nullable is described as being useful only for `@Inject`ed  fields / injection points. So it does not make too much sense to use it here, does it?

Reply:
I guess I could just document that it can return nulls then